README.md

```diff
@@ -17,7 +17,7 @@ markdown file.
 
 From class lecture slides/video we discussed that the mergesort run time will be $\Theta(nlog(n))$. With this knowledge, I continued on from the proof explained in the slides and understood that since there is a nested loop this changes the complexity to $\Theta(n^2 log(n))$. This new complexity would also be the worst case of runtime complexity after combinging the implementation of the outer loop, the inner loop, and the merge function itself. 
 
-I used lecture slides and the videos provided from class. I also used geeksforgeeks: https://www.geeksforgeeks.org/time-and-space-complexity-analysis-of-merge-sort/ to help explain the overall time complexity of merge sort. I jumped to this exercise because I had just worked on divide and conquer so I also used some logic and resources previously referrenced in that exercise.
+I used lecture slides and the videos provided from class. I also used geeksforgeeks: https://www.geeksforgeeks.org/time-and-space-complexity-analysis-of-merge-sort/ to help explain the overall time complexity of merge sort. I jumped to this exercise because I had just worked on divide and conquer so I also used some logic and resources previously referrenced in that exercise. I communicated and collaborated with fellow students Daniel Collins and Will Greiner. Their repositories are mergesort-Will-GreinerUWYO and mergesort-22dcolli-1.
```

Comment:
Can you please clarify to what extent you used code provided by them?

Reply:
Me an Will talked after class about how to approach this problem so I don't have use from his. Daniel had helped me through this and explained his code implementation to me which I understood the best, so I believe that our implementations are very similar. I know he has two repositories from his previous attempt at this class, I have his current one referenced. I can go and look through his references as well since if he used others work then mine would be similar to others as well, but I can modify my references to make it clearer.